### ISSUE TITLE
There is no template at js/tmpl/undefined.ejs

### ISSUE THREAD

**Issue by ZyanKLee** [deadtrickster/prometheus_rabbitmq_exporter#1]
when enabling rabbitmq_management_metrics on rabbitmq 3.6.2 the management webinterface complains about ```undefined: There is no template at js/tmpl/undefined.ejs```

in the firefox dev tools I can see, that some where in the request a ressource "undefined" is requested and not found.

those are the plugins present on this system:
```
 Configured: E = explicitly enabled; e = implicitly enabled
 | Status:   * = running on rabbit@levigo-rmq
 |/
[e*] amqp_client                       3.6.2
[  ] cowboy                            1.0.3
[  ] cowlib                            1.0.1
[e*] mochiweb                          2.13.1
[  ] prometheus                        0.1.1
[  ] rabbitmq_amqp1_0                  3.6.2
[  ] rabbitmq_auth_backend_ldap        3.6.2
[  ] rabbitmq_auth_mechanism_ssl       3.6.2
[  ] rabbitmq_consistent_hash_exchange 3.6.2
[  ] rabbitmq_event_exchange           3.6.2
[  ] rabbitmq_federation               3.6.2
[  ] rabbitmq_federation_management    3.6.2
[E*] rabbitmq_management               3.6.2
[e*] rabbitmq_management_agent         3.6.2
[  ] rabbitmq_management_metrics       3.6.2
[  ] rabbitmq_management_visualiser    3.6.2
[  ] rabbitmq_mqtt                     3.6.2
[  ] rabbitmq_recent_history_exchange  1.2.1
[  ] rabbitmq_sharding                 0.1.0
[  ] rabbitmq_shovel                   3.6.2
[  ] rabbitmq_shovel_management        3.6.2
[  ] rabbitmq_stomp                    3.6.2
[  ] rabbitmq_tracing                  3.6.2
[e*] rabbitmq_web_dispatch             3.6.2
[  ] rabbitmq_web_stomp                3.6.2
[  ] rabbitmq_web_stomp_examples       3.6.2
[  ] sockjs                            0.3.4
[e*] webmachine                        1.10.3
```

the bug does not appear when enabling prometheus only - but it does as soon as I enable rabbitmq_management_metrics. And it does so no matter if I have prometheus enabled or disabled.

**Comment by deadtrickster** [comment]
Hey, thank you for the report!
I understand why it's tied to rabbitmq_managements_metrics - management plugins are to inject stuff to http endpoint and gui. Maybe I did something wrong here. Can you confirm metrics collection and exposure via /api/metrics still works?

**Comment by ZyanKLee** [comment]
yes, that works.

**Comment by deadtrickster** [comment]
related: https://github.com/rabbitmq/rabbitmq-management/issues/251

**Comment by deadtrickster** [comment]
@ZyanKLee Thank again for the report. This will be fixed in the upcoming RabbitMQ 3.6.4 release.
